trulens_eval/examples/experimental/virtual_example.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        virtual_recorder.add_record(rec)
</pre><p>By adding the records with the virtual_recorder, assuming this kicks off feedback generation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Yes. Will rename and emphasize.

Reply:
Did not rename but added more docs and feedback_mode argument to run in particular feedback mode for the given record.